src/entities/tag.ts

```diff
@@ -0,0 +1,33 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToMany } from 'typeorm'
+import { Card } from './card'
+import { Board } from './board'
+
+@Entity()
+export class Tag {
+// ------------------------------------
+// =        ENTITY DEFINITION
+// ------------------------------------
+    @PrimaryGeneratedColumn()
+    id: number
+
+    @Column({
+        type: 'text',
+        length: 25
+    })
+    label: string
+
+    @Column({
+        type: 'text',
+        length: 25
+    })
+    color: string
```

Comment:
Is it possible to have an Enum ?

Reply:
Je stocke direct les codes Hexa ?

Reply:
Done, added the color names associated to their hexa codes. I also added enums for roles